roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Assert st2chatops_hubot_adapter is specified
+  fail:
+    msg: >
+      '"st2chatops_hubot_adapter" must be supplied in roles/st2chatops/defaults/main.yml for st2chatops installation.
```

Comment:
We can omit mentioning `roles/st2chatops/defaults/main.yml` since it's not single point of truth.
Just asking user to provide `{{ st2chatops_hubot_adapter }}` var is enough.